docs/zkapps/o1js/interact-with-mina.md

```diff
@@ -171,11 +233,11 @@ console.log(tx.toPretty());
 
 :::info
 
-MINA amounts, in all o1js APIs and elsewhere in the protocol, are always denominated in nanoMINA = `10^(-9)` MINA. This is why we set `const MINA = 1e9`.
+MINA amounts, in all o1js APIs and elsewhere in the protocol, are always denominated in nanoMINA = `10^(-9)` MINA (which is why we set `const MINA = 1e9`).
```

Comment:
what does this mean? do we need to include it? and can we link somewhere to explain it?

> (which is why we set `const MINA = 1e9`).